plugins/out_s3/s3.c

```diff
@@ -1126,6 +1154,24 @@ static int s3_put_object(struct flb_s3 *ctx, const char *tag, time_t create_time
     return -1;
 }
 
+int s3_body_md5(char *body, size_t body_size, char *body_md5, size_t body_md5_size)
```

Comment:
this function calculates the md5 hash right? I find the function name to be confusing. Can you make it something clear to what it does, something like, `get_body_md5`